packages/common/src/content/compose.ts

```diff
@@ -219,6 +219,9 @@ const getMetadataUpdatedDateInfo = (item: IItem, metadata?: any) => {
 };
 
 // public API
+// AGO has a util for determining display name and item type icons
+// that we can use for reference
+// https://devtopia.esri.com/WebGIS/arcgis-app-components/blob/master/src/components/arcgis-item-type/utils.ts
```

Comment:
I'd say we stick with our getContentTypeIcon for now for consistency. I added this comment here because I just got that info from the AGO team, it was new to me as well. I think this needs a holistic review by Design first.